documentation/asciidoc/computers/ai/getting-started.adoc

```diff
@@ -154,8 +151,40 @@ This demo performs 17-point human pose estimation, drawing lines connecting the
 
 [source,console]
 ----
-$ rpicam-hello -t 0 --post-process-file ~/rpicam-apps/assets/hailo_yolov8_pose.json --lores-width 640 --lores-height 640
+$ rpicam-hello -t 0 --post-process-file /usr/share/rpi-camera-assets/hailo_yolov8_pose.json
+----
+
+=== Alternative Package Versions
+
+The AI Kit/HAT+ devices will not function if there is a version mismatch between the Hailo software packages and device drivers. In addition, Hailo's neural network tooling may also require particular a version for the generated model file. At the time of writing, the latest package version is 4.19, but if earlier versions are required, the following steps should be followed:
+
+. If you have previously used `apt-mark` to hold any of the relevant packages, you may need to unhold them:
```

Comment:
Is this something that a typical user is _likely_ to have done?

Reply:
Only if they have an explicit reason to do so, e.g. running older models.